contrib/pyln-proto/requirements.txt

```
@@ -1,6 +1,6 @@
 base58 ~= 2.0.1
 bitstring ~= 3.1.6
-coincurve ~= 13.0
+coincurve ~= 15.0.1
```

What is the error that you receive here? can you add a little bit more content?

```suggestion
coincurve >= 13.0
```

However, we preferer to use the `>=` instead of the `~=`, because  `~=` will postpone in the future a future error.

---

It was because I didn't have `libsecp256k1-dev`. Added to docs.